cpp/doxygen/developer_guide/JIT_LTO.md

```
@@ -0,0 +1,989 @@
+
+```c++
+template <typename T, T Divisor>
+__device__ bool is_divisible_impl(T value)
+{
+  return value % Divisor == 0;
+}
+```
+
+Here is an example of an adapter file that contains an adapter function:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)